index.html

```diff
@@ -34,5 +35,156 @@ <h1 class="me">Hey there. I’m Mariane
         </ul>
         <a href=""></a><img src="image/Action.png" alt="action-button" id="more">
     </section>
+
+    <section id="recent-works">
+        <h2 class="">My Recent Works</h2>
+        <img src="image/Indicator.png" alt="Indicator" id="indicator">
+        <section class="box-articles">
+            <article class="box">
+                <section class="in-box">
+                    <h3 class="">Multi-Post Stories Gain+Glory</h3>
+                    <div class="btn-language 1 ">
+                        <button type="button">Ruby on rails</button>
+                        <button type="button">css</button>
+                        <button type="button">JavScript</button>
+                        <button type="button">html</button>
+                    </div>
+                    <button type="button" class="btn-big">See Project</button>
+                </section>
+            </article>
+            <article class="box">
+                <section class="in-box">
+                    <h3 class="">Multi-Post Stories Gain+Glory</h3>
+                    <div class="btn-language 1 ">
+                        <button type="button">Ruby on rails</button>
+                        <button type="button">css</button>
+                        <button type="button">JavScript</button>
+                        <button type="button">html</button>
+                    </div>
```

Comment:
You have a beautiful design. However, you'll agree that the list of project technologies is just that - a list. I think that using an `li` tag for each item and a `ul` tag to enclose them all will be more semantic.

Remember to modify it in all the projects.